tests/acceptance/expected-failures-with-oc10-server.md

```diff
@@ -0,0 +1,8 @@
+## This file contains the scenarios from web tests that are currently expected to fail on oC10
```

Comment:
We can now be more flexible here about writing text. Here is some suggested description for the top of this file:

```suggestion
## Scenarios from web tests that are expected to fail on oC10

Lines that start with a format like "- [something.feature:n]" are lines that document a specific expected failure. Follow that with a URL to the line in the feature file in GitHub. Please follow this format for the actual expected failures. 

Level-3 headings should be used for the references to the relevant issues. Include the issue title with a link to the issue in GitHub.

Other free text and markdown formatting can be used elsewhere in the document if needed. But if you want to explain something about the issue, then please post that in the issue itself.

```

You could put similar in the other expected-failures file also.